### PR TITLE
NoReplyMention: add option to only exclude specific users from pings

### PR DESCRIPTION
**Pull request by outfoxxed**
Adds a new behavior option to NoReplyMention that lets you avoid pinging only specific users instead of only not pinging specific users.

Note: Not sure if I was supposed to add myself to the author list or not, looking through commit history it seems like I am.

### CODE REVIEW

**Comment by Vendicated** [comment]
it's a bit hard to follow for me what this is trying to accomplish. isn't this already a feature if you enable the invert shift behaviour option?

**Comment by outfoxxed** [comment]
Looks like it sorry. Haven't updated in a while so its not in my main browser, and misunderstood what it was doing while writing the pr.

**Comment by Vendicated** [comment]
ooo no need to apologise, thanks for your work anyway!

**Comment by outfoxxed** [comment]
Actually thats not quite right. Inverting shift just makes shit clicking the reply button not work. I'll check how the options interact and see if I need to change anything (I assume it works the way it does for a reason but idk)

**Comment by outfoxxed** [comment]
This pr seems to work fine in its current state with inverseShiftReply as mentioned on the discord, and does not overlap feature wise.
